### DISCUSSION TITLE
Can SDMaid SE coexist with SDMaid first edition?

### DISCUSSION THREAD

**Discussion by sirleon23** [d4rken-org/sdmaid-se#631] [Q&A] [answered]
Hi guys, just a question, can this new app coexist with the previous version? I'd not like to uninstall the first application until the second one is released as stable

**Comment by d4rken** [comment, marked as answer]
Yes they can co-exist. There is no problem.
But try to not let both perform actions at the same time, results could be incomplete.

**Reply by sirleon23** [reply]
So no scheduled action in the same time (like clean uninstalled apk)
